asciidoc/courses/aura-dashboards/modules/1-getting-started/lessons/3-exploring-dashboards/lesson.adoc

```diff
@@ -0,0 +1,148 @@
+
+You can copy and paste the script into the Query Editor, and click the **Run** button to execute it.
+
+
+image::images/run-query.png[The import-recommendations.cypher script in the Query Editor]
+
+Once the script has finished running, you will see a summary of the nodes and relationships created in your Aura instance. 
+
+Go to the Explore tab to visualize and explore the data you have just loaded: 
+
+
+image::images/explore-check.png[The Explore tab showing the loaded data graph]
+
+//TO-DO: Use this content later when we have cloud import credential rotation available
+
+// == Importing data from cloud
+
+// Importing data from cloud storage services is a convenient way to manage large datasets, and to ensure data security and accessibility.
+
+// When you go to the Import menu, you will see an option to add a new data source: 
+
+
+// image::images/import-menu.png[The data source importing UI]
+
+// In this example, you will learn how to import data from an S3 bucket. If you have other cloud storage services, the steps are similar:
+
+// * Set up access credentials: You might need to create an access key and secret key to authenticate your access to the cloud storage service. If you are using AWS S3, you can create an IAM user with the necessary permissions to access the S3 bucket.
+// * Configure the data source in Data Importer: In the Data Importer interface, select the option to add a new data source and choose the cloud storage service you are using.
+
+
+// image::images/new-data-source.png[The data source importing options]
+
+
+// * You will need to provide the access credentials and the path to the data files, and make sure the bucket and files are accessible from Data Importer.
+
+// To import data from an S3 bucket, you will need to provide the following information:
+
+// * Access Key: Your AWS access key ID.
+// * Secret Key: Your AWS secret access key.
+
+// image:://images/import-aws.png[The AWS S3 import configuration dialog]
+
+// * Bucket Name: The name of the S3 bucket containing the data files.
+// * File Path: The path to the data files within the S3 bucket.
+
+
+
+
+// image:://images/import-aws-2.png[The AWS S3 import configuration dialog continued]
+
+== Generating a demo dashboard using AI
+
+
+Once you have successfully loaded the data into your Aura instance, you can generate a demo dashboard using the AI tool.
+
+You will learn more about creating and managing dashboards in the next lessons, but for now, let's see how to quickly generate a dashboard using AI. 
+
+Prior to using the AI tool, ensure that you have read the AI disclaimer available in the Aura Dashboards interface: 
+
+image::images/genai-disclaimer.png[AI disclaimer dialog,width=600,align=center]
+
+From the Dashboards menu, click on the **Create dashboard** dropdown, and select **Create with AI**.
```

Comment:
If it's the first time a user does this, they'll have an empty state in Dashboards and are presented with _Create with AI_, _Create from scratch_, or _Import_ and won't have to use the dropdown.